docs/LV_client/examples/mono1_pilot_example.md

```diff
@@ -0,0 +1,64 @@
+# mono1_pilot_example.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono1_pilot_example.png"/>
+</p>
+
+### Description
+
+This example shows how to use the monoDrive client to configure the monoDrive simulator in closed loop mode with all the sensors.
+
+For technical support contact us at <b>support@monodrive.io</b> 
+
+### Inputs
+
+- **Throttle:**  Change the vehicle's throttle (0 - 1)
+ 
+
+- **Steering:**  Move to drive the car right and left (-1 to 1)
```

Comment:
While using the active voice, I think this may flow a little better: "While driving, move the vehicle right and left (-1 to 1)"